components/table/table-wrapper.js

```diff
@@ -589,6 +612,30 @@ export class TableWrapper extends RtlMixin(PageableMixin(SelectionMixin(LitEleme
 		}
 	}
 
+	_updateStickyAncestor(node, popoverOpened) {
+		if (!this.stickyHeaders) return;
+
+		node = getComposedParent(node);
```

Comment:
This code basically walks up the tree to find the sticky element and keeps count of the popovers within, which is targeted by the CSS to elevate the `z-index`.

For now, I think we can only expect one dropdown within a table header cell. However, more than one is entirely possible in the future, for example if menu-items supported tooltips (something Jeff and I chatted about recently).